extensions/mssql/src/objectManagement/ui/restoreDatabaseDialog.ts

```diff
@@ -179,16 +201,18 @@ export class RestoreDatabaseDialog extends ObjectManagementDialogBase<Database,
 	private createRestoreInfo(): azdata.RestoreInfo {
 		const isRestoreFromBackupFile = this.restoreFrom.value === localizedConstants.RestoreFromBackupFileOptionText;
 		let options: { [key: string]: any } = {};
-		Object.entries(this.objectInfo.restorePlanResponse.planDetails).forEach(([key, value]) => {
+		Object.entries(this.objectInfo.restorePlanResponse?.planDetails).forEach(([key, value]) => {
```

Comment:
Looks like Object.entries throws an error if it's passed an undefined value. Could you add a check for that here?

Reply:
Issue was caused by intercahnging the restroeFrom dd option and it is resolved now.